### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Ramalaxmii**
fghdcgfhfgnbcvbncvb

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ First off, thank you for taking the time to contribute! :+1: :tada:
 
 ### Code of Conduct
 
-This project is governed by the [Spring Code of Conduct](CODE_OF_CONDUCT.adoc).
+This is a project is governed by the [Spring Code of Conduct](CODE_OF_CONDUCT.adoc).
```
Comment (Ramalaxmii) [inline CONTRIBUTING.md:19]: test

Comment (Ramalaxmii) [reply CONTRIBUTING.md:19]: fdsds

**Review by Ramalaxmii — COMMENTED** [review]
tyerty

**Comment by bclozel** [comment]
@Ramalaxmii please use your own fork for tests. This repository is watched by thousands of people and they're all getting notifications about this.

**Pull request closed without merging**